src/main/java/de/doubleslash/keeptime/view/SettingsController.java

```diff
@@ -164,7 +176,7 @@ private void initialize() {
          controller.updateSettings(hoverBackgroundColor.getValue(), hoverFontColor.getValue(),

```

Comment:
we should pass an object as param instead of all values on each own. Like for editWork passing a Settings object with the updated Settings as param, instead of each param of settings on each own

Reply:
changed in #49